signhandler/src/main/java/se/swedenconnect/signservice/signature/tbsdata/impl/PDFTBSDataProcessor.java

```diff
@@ -141,9 +141,12 @@ protected void checkToBeSignedData(@Nonnull final byte[] tbsData, final boolean
             throw new InvalidRequestException("Signing time is not allowed in PAdES requests in strict processing");
           }
         }
+        // Test if signing time is current
+        this.checkSigningTime(Objects.requireNonNull(getCmsSigningTime(signedAttributes),
+          "Null signing time with present signing time attribute").toInstant());
       }
     }
-    catch (final IOException e) {
+    catch (final IOException | IllegalArgumentException e) {
```

Comment:
See above

Reply:
This is actually shown by bouncy castle, I have updated this to also cover a possible NullPointerException.